src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -60,6 +60,7 @@ log_level: INFO
 - `ol` - **Optional**. Output layer layout.
 - `iml` - **Optional**. Input model layout.
 - `oml` - **Optional**. Output model layout.
+- `shape` - **Optional**. Input layer shape.
```

Comment:
Description doesn't really reflect what it does.

Reply:
I will add a better description.

Reply:
Done